lib/Doctrine/Common/Proxy/ProxyGenerator.php

```diff
@@ -559,18 +564,16 @@ private function generateMagicSet(ClassMetadata $class)
             return '';
         }
 
-        $inheritDoc = $hasParentSet ? '{@inheritDoc}' : '';
-        $magicSet   = sprintf(<<<'EOT'
+        $magicSet = <<<EOT
     /**
-     * %s
-     * @param string $name
-     * @param mixed  $value
+     * $inheritDoc
```

Comment:
I suppose you meant 
```suggestion
     * {@inheritDoc}
```

Reply:
As with the `generateMagicGet()` implementation, the `$inheritDoc` is set to `{@inheritDoc}` if `$reflectionClass->hasMethod('__set')` evaluates to `true`, see line 556.